src/bidmanager.js

```diff
@@ -61,8 +61,10 @@ exports.bidsBackAll = function() {
   return bidsBackAll();
 };
 
-function getBidSetForBidder(bidder) {
-  return $$PREBID_GLOBAL$$._bidsRequested.find(bidSet => bidSet.bidderCode === bidder) || { start: null, requestId: null };
+function getBidSet(bidder, adUnitCode) {
+  return $$PREBID_GLOBAL$$._bidsRequested.find(bidSet => {
+    return bidSet.bids.filter(bid => bid.bidder === bidder && bid.placementCode === adUnitCode).length > 0;
```

Comment:
This will work with the approach we are now taking with queued bid requests -- bid requests and responses will be removed from the `_bidsRequested` and `_bidsReceived` arrays when that placement is part of the outgoing bid request.